assets/css/appV2/table.scss

```diff
@@ -12,6 +12,24 @@ th:last-child{
   border-radius: 0 10px 10px 0;
 }
 
+.cardShare::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 5px;
+  border-radius: 0 0 0 30px;
+  background-color: $primary;
+}
+.shared::before {
+  background-color: $green;
+}
```

Comment:
I just realized that it's not our custom theme colors. You have to do it this way `theme-color('green')`, or you'll get bootstrap primary and green colors